manifests/s/SecTeerApS/SecTeerVulnDetect/2.4.2.0/SecTeerApS.SecTeerVulnDetect.locale.en-US.yaml

```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: SecTeerApS.SecTeerVulnDetect
+PackageVersion: 2.4.2.0
+PackageLocale: en-US
+Publisher: SecTeer ApS
+PackageName: SecTeer VulnDetect
+License: Proprietary, see license file for details. Free for Personal / home use.
+Copyright: © SecTeer ApS. All rights reserved.
+ShortDescription: 'VulnDetect detects and silently updates third party applications on Windows. Use: winget install SecTeerApS.SecTeerVulnDetect --override "/VERYSILENT /email=your@email.tld"'

```

Comment:
Applications are required to install unattended by default. Since this package requires the email to be specified, it cannot be added until interactive-only support is added.

![image](https://user-images.githubusercontent.com/12611259/214136233-71f80fe7-fc70-4213-b936-99e91665ca20.png)

Reply:
@msftbot interactive-only